corehq/apps/export/models/new.py

```diff
@@ -792,6 +800,76 @@ def _new_from_schema(cls, schema):
         )
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = SMS_EXPORT
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        return cls(
+            domain=schema.domain,
+            tables=[
+                TableConfiguration(
+                    label="Messages",
+                    columns=[
+                        RowNumberColumn(),
+                        ExportColumn(
+                            label="Contact ID",
+                            item=ExportItem(
+                                path=[PathNode(name='couch_recipient')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Timestamp",
+                            item=ExportItem(
+                                path=[PathNode(name='date')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Username",
+                            item=ExportItem(
+                                path=[PathNode(name='couch_recipient')],
+                                transform=USERNAME_TRANSFORM,
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Phone Number",
+                            item=ExportItem(
+                                path=[PathNode(name='phone_number')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Direction",
+                            item=ExportItem(
+                                path=[PathNode(name='direction')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Message",
+                            item=ExportItem(
+                                path=[PathNode(name='text')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Type",
+                            item=ExportItem(
+                                path=[PathNode(name='workflow')]
```

Comment:
This will need a new transform to implement this https://github.com/dimagi/commcare-hq/blob/je/sms-export/corehq/apps/reports/standard/sms.py#L289-L305